genref/output/md/kubeadm-config.v1beta3.md

```diff
@@ -264,109 +264,6 @@ node only (e.g. the node ip).</p>
   
     
 
-## `BootstrapToken`     {#BootstrapToken}
-    
```

Comment:
:wave: @tengqm Are `BootstrapToken` and `BootstrapTokenString` replaced by similar content later in the file?
It is difficult to identify the individual changes.

Reply:
These are generated contents, I didn't verify the changes line by line.
As for the content, both `BootstrapToken` and `BootstrapTokenString` look consistent.